hathor/consensus.py

```diff
@@ -253,7 +253,7 @@ def update_voided_info(self, block: Block) -> None:
                     assert best_block.hash is not None
                     best_meta = best_block.get_metadata()
                     self.log.debug('index previous best block', height=best_meta.height, block=best_block.hash_hex)
-                    storage.add_new_to_block_height_index(best_meta.height, best_block.hash)
```

Comment:
Is it a bug fix?

Reply:
I don't know why I removed this line. I'll review this more carefully.

Reply:
Oh, this isn't a remove, it's a replace. Github line comments aren't great so see what's bellow the diff. I usually try to comment on the end of a block of changes so it's easier to see. Anyway I'm running the tests again with the old method call, but I think this had to do with the loading transactions, where this call can be a reorg sometimes.

Reply:
I think this should actually me removed, instead of replaced. But it needs some more thought.

Reply:
I guess you moved this change to the next PR, right?

Reply:
Reverted.